cli/user/src/main/java/io/pravega/cli/user/UserCLIRunner.java

```diff
@@ -23,6 +23,7 @@
 import io.pravega.cli.user.config.InteractiveConfig;
 import lombok.Cleanup;
 import lombok.val;
+import org.junit.Assert;
```

Comment:
There are 2 unused import because of which build is failing `import org.junit.Assert;` and `import ch.qos.logback.classic.Level;` . 

Changes LGTM.

Reply:
Thanks @ShwethaSNayak.  New commit pushed to fix this.